now.json

```
@@ -445,6 +445,10 @@
       "source": "/guides/nextjs(.*)",
       "destination": "/guides/tina-cloud/add-tinacms-to-existing-site/overview/"
     },
+    {
+      "source": "/guides/tinacms(.*)",
```

We had this before because we had no remaining guides in tinacms (There was a single guide in "guides/tina-cloud"). 
I think you'll want 
```suggestion
      "source": "/guides/tina-cloud(.*)",
```